.github/workflows/build-packages.yml

```diff
@@ -830,27 +830,59 @@ jobs:
       - uses: hdl/conda-ci@master
 
   #59
-  uhdm-integration-verilator-linux:
+  uhdm-integration-verilator-linux-py37:
```

Comment:
I feel like this style be better done with a matrix?

Reply:
It was done this way because you can't set name of a job and its required jobs (`needs`) based on a matrix variable.

Therefore using a matrix some package using Python 3.8 can fail because its required job has a matrix to build packages using multiple versions of Python and any of them failed, even when Python 3.8-based package was successfully built. I'm not sure if the slightly better style is worth it.

The workflow file is huge and complex nevertheless. I think the only real improvement would be to create `build-packages.yml` with a script based on some compact job definitions file.